test/runtests.jl

```diff
@@ -1,4 +1,5 @@
-using ParallelTestRunner: runtests
+using ParallelTestRunner: runtests, parse_args
+using Base.Filesystem: path_separator
```

Comment:
This is unused or am I missing something?

Reply:
oop, leftover from workaround. thx!

Reply:
d008993